tests/helpers/utils_for_testing.py

```diff
@@ -67,8 +84,8 @@ def assert_flattened_outcome_as_expected(
             column of the flattened df.
         resolve_multiple (Callable): How to handle multiple values within the lookahead window.
             Takes a a function that takes a list as an argument and returns a float.
-        values_colname (str, optional): Column name for the new values. Defaults to "val".
-        fallback (List, optional): What to fill if no outcome within lookahead days. Defaults to 0. # noqa: DAR101
+        values_colname (str): Column name for the new values. Defaults to "val".
+        fallback (List): What to fill if no outcome within lookahead days. Defaults to 0. # noqa: DAR101
```

Comment:
```suggestion
        fallback (Optional[List]): What to fill if no outcome within lookahead days. Defaults to 0.
```

Reply:
This seems like darglint informed you about this?

Reply:
Regardless the type hint and text are wrong here.  It does not default to 0 it defaults to np.nan.

Reply:
The same is the case for: assert_flattened_values_as_expected

Reply:
I believe it should be `Union[float, np.nan]`?

Reply:
Ah, yeah, `Optional` was from when I didn't understand it, and have probably failed to update the docstring.

It can take any value that can go in a pandas df. Have added a few.

Reply:
The noqa?
> noqa: DAR101

Reply:
Was from formatting errors on my part. Fixed now.